src/servicetask/servicetaskenvironment.ts

```diff
@@ -29,7 +29,7 @@ export interface IServiceTaskInstances {
   uploadAttachment(processId: string, instanceId: string, fileName: string, dataBase64: string): Promise<string>;
   generateInstanceReport(instanceIds: string[], draftId: string, type: IGenerateReportRequestType): Promise<{ doc: string /* Base64*/; fileName: string }>;
   executeInstance(processId: string, instance: IInstanceDetails, startEventId?: string, accessToken?: string): Promise<string>;
-  getSupervisor(userId: string, loadSupervisor?: boolean): Promise<string>;
+  getSupervisor(userId: string): Promise<IUserDetails | string>;
```

Comment:
should be added to IServiceTaskRoxApi, not to IServiceTaskRoxApi. IServiceTaskInstances contains the instance API methods.

Reply:
Moved getSupervisor method to IServiceTaskRoxApi